topics/climate/tutorials/fates/slides.html

```diff
@@ -0,0 +1,217 @@
+---
+layout: base_slides
+logo: "GTN"
+title: "Functionally Assembled Terrestrial Ecosystem Simulator (FATES)"
+questions:
+  - What is FATES?
+  - How can I use FATES?
+objectives:
+  - Have an understanding of what FATES is and how it works
+time_estimation: "30m"
+key_points:
+  - FATES
+contributors:
+  - rosiealice
+  - annefou
+tags:
+  - climate
+  - ecology
+requirements:
+  - type: "internal"
+    topic_name: introduction
+    tutorials:
+      - galaxy-intro-101-everyone
```

Comment:
I am not sure how to do it. I tried:

```
- type: "internal"
    topic_name: climate
    slides:
      - introduction
```

but I can't see them. Do you know the syntax to use for adding slides in requirements?

Reply:
no, sorry :(

Reply:
Otherwise, I can add it as external:
```
 - type: "external"
    title: introduction to climate data
    link: https://training.galaxyproject.org/training-material/topics/climate/slides/introduction.html#1

```

Reply:
that could do the trick I guess :)

Reply:
Anne you can also look at the overall metadata file: https://github.com/galaxyproject/training-material/blob/master/topics/transcriptomics/metadata.yaml#L8

Reply:
Ah yes! So this is where we set the default requirements... I did not know about it!!! Thanks @bgruening